protocol/x/stats/types/tx.go

```diff
@@ -10,5 +12,15 @@ func (msg *MsgUpdateParams) GetSigners() []sdk.AccAddress {
 }
 
 func (msg *MsgUpdateParams) ValidateBasic() error {
+	if _, err := sdk.AccAddressFromBech32(msg.Authority); err != nil {
```

Comment:
unit tests?

Reply:
There are no existing unit tests for this msg, but can add.

Reply:
For these files that have no unit tests, my plan was to defer that to the auditor. That ok?